### DISCUSSION TITLE
Does WinGet UI have the ability to scan updates for programs not installed & portable .exe's

### DISCUSSION THREAD

**Discussion by jackuars7** [Devolutions/UniGetUI#1433] [Q&A] [answered]
Hi all, 

I have been a longtime user of SUMo, and unfortunately, it has reached its End of Life. The developer has recommended some alternatives, and that's how I ended up here. One thing that I really miss in SUMo, which most other updaters don't have, is the ability to scan for folders that contain software installers or portable exe's, which helps in finding their updates as well.

As a newcomer to Winget UI, I wanted to know if this software provides the ability to do the same?

Thanks!

**Comment by ContinuumOperand** [comment, marked as answer]
Not directly what you asked, and I believe someone can give a better reply for that, but for portable installs it might be worth migrating your software to a different source, personally I prefer to use Scoop for it (most things there are portable by nature, unless you're using the non-portable bucket). Chocolatey also offers some portable versions of apps as alternatives to conventional installations... Winget itself still hasn't embraced portable installations from my experience.

The good news is that, of course, WingetUI supports all of those sources so you can use all of them where each does the best job for your needs, and still have a unified interface and experience to manage your software.

About scanning folders and updating manually installed portable software I don't think that's a thing, you'd need to reinstall your portable software with one of the supported package managers to keep it tracked...

**Reply by marticliment** [reply]
Yes, @continuumoperand is right. WingetUI does not scan for updates on portable apps (WingetUI does not directly scan for updates, package managers do so). However, you can install _portable_-like apps from scoop and chocolatey, package managers supported on WingetUI
